exec/totemconfig.c

```diff
@@ -899,9 +900,10 @@ static int check_for_duplicate_nodeids(
 			continue;
 		}
 
-		if (strcmp(tmp_key, "nodeid") != 0) {
+		if (last_node_pos == node_pos) {
```

Comment:
Could you please add short comment about rely on cmap order (something like "code relies on fact that icmap keys are returned ordered")?